src/main/scala/io/iohk/ethereum/blockchain/sync/regular/BlockImporter.scala

```diff
@@ -63,10 +63,10 @@ class BlockImporter(
   private def running(state: ImporterState): Receive = handleTopMessages(state, running) orElse {
     case ReceiveTimeout => self ! PickBlocks
 
-    case PrintStatus => log.info("Block: {}, is on top?: {}", blockchain.getBestBlockNumber(), state.isOnTop)
+    case PrintStatus => log.info("Block: {}, is on top?: {}", blockchain.getBestBlock().get.number, state.isOnTop)
 
     case BlockFetcher.PickedBlocks(blocks) =>
-      SignedTransaction.retrieveSendersInBackGround(blocks.toList.map(_.body))
+//      SignedTransaction.retrieveSendersInBackGround(blocks.toList.map(_.body))
```

Comment:
this is only for caching, right?

Reply:
yes, it is. Would add it back when this go to staging